cmd/virt-v2v-monitor/virt-v2v-monitor.go

```diff
@@ -28,7 +29,7 @@ func LimitedScanLines(data []byte, atEOF bool) (advance int, token []byte, err e
 	if token != nil || err != nil {
 		return
 	}
-	if len(data) == bufio.MaxScanTokenSize {
+	if len(data) == 1*1024*1024 {
```

Comment:
same regarding the `1*`, also, i see `1024*1024` 3 times already. maybe worth a const?